src/components/TableHeaderCell/TableHeaderCell.tsx

```diff
@@ -82,13 +82,9 @@ export const TableHeaderCell = ({
       id={id}
       {...other}
     >
-      <Button
-        variant="table-header"
-        iconPosition="after"
-        iconName="arrow-narrow-down"
-        onClick={onClick}
-      >
-        {children}
+      <Button variant="table-header" onClick={onClick}>
+        {text}
```

Comment:
The change here from `children` to `text` is cleaning up a mistake from when I renamed the `text` prop in `Button` to `children`.